ttnn/cpp/ttnn/device.cpp

```diff
@@ -9,7 +9,7 @@ namespace ttnn {
 namespace device {
 
 Device &open_device(int device_id, size_t l1_small_size, size_t trace_region_size) {
-    tt::DevicePool::initialize({device_id}, 1, l1_small_size, trace_region_size);
+    tt::DevicePool::initialize({device_id}, 1, l1_small_size, trace_region_size, {});
```

Comment:
I think you can revert this change and constructor default value take care of it

Reply:
shoot, missed this. I should be coming back to this initialize call soon. I'm planning to remove the l1_bank_remap altogether from device, and that should clean this up as well.